### ISSUE TITLE
Very long loading time (Ubuntu)

### ISSUE THREAD

**Issue by hotice** [Aluxian/Whatsie#340]
* Operating system: Ubuntu 16.04 (64bit)

Using the latest Whatsie dev (2.0.18), the app takes about 5 minutes to load (it displays a blank window with a spinning circle). I've tested multiple Whatsie versions and this does not occur with versions: 2.0.12, 2.0.13, 2.0.14. It occurs with 2.0.15 and 2.0.18 (I didn't test the versions in between).

I have no issues with WhatsApp Web used in a web browser.

Log attached.

Edit: after the initial load, the app continues with this behavior the next time is ran, but now it displays the "Connecting to WhatsApp" message instead of a blank loading window:

![whatsie-bug](https://cloud.githubusercontent.com/assets/325449/16987403/c2849562-4e94-11e6-9ee2-3e38d9b734cf.png)

Log attached.
[1469019577700.txt](https://github.com/Aluxian/Whatsie/files/373721/1469019577700.txt)

**Comment by Aluxian** [comment]
Nothing seems wrong in the log... I'm clueless. Actually, there are some logs missing in your file. I think there's something wrong with the logger, I'll check that first.

**Comment by Aluxian** [comment]
Can you please attach another log running 2.0.19?

**Comment by hotice** [comment]
Sure
[1469439805038.txt](https://github.com/Aluxian/Whatsie/files/381177/1469439805038.txt)

**Comment by hotice** [comment]
I just booted an Ubuntu 16.04 64bit LiveUSB on the same computer on which I experienced this issue, and the app worked without any issues. So I guess there's something on my computer that prevents it from working correctly. Of course, it's still a bug, because like I mentioned in the initial bug report, it doesn't occur with versions prior to 2.0.14 for me. But at least it probably doesn't affect many (hopefully).
